Sources/TuistKit/Generator/ManifestLinter.swift

```diff
@@ -0,0 +1,40 @@
+
+import Foundation
+import ProjectDescription
+import TuistCore
+
+protocol ManifestLinting {
+    func lint(project: Project) -> [LintingIssue]
+}
+
+class ManifestLinter: ManifestLinting {
```

Comment:
I wonder if we need to have some linting at the Graph level which notifies the user of configurations not defined in manifests. I can imagine on larger projects it might be easy to miss one, or even worse, mis-spell one.

Reply:
👍 
There's a scheme linter in place too to verify this

https://github.com/tuist/tuist/pull/451/files#diff-6a1a7d8fd087d39e1cf8d85558ece81c

Reply:
Added https://github.com/tuist/tuist/issues/467 for a follow up